Source/WTF/wtf/PlatformHave.h

```diff
@@ -1818,6 +1818,16 @@
 #define HAVE_NSRESPONDER_WRITING_TOOLS_SUPPORT 1
 #endif
 
+#if !defined(HAVE_WRITING_TOOLS) \
+    && __has_include(<WritingTools/WritingTools.h>) \
```

Comment:
Do we really need this has_include check? WritingTools.framework should be in all the SDKs we build with since it shipped last year.
```suggestion
```

Reply:
I will double check, I'm not 100% sure now, but I believe it was failing to compile in EWS without it

Reply:
Indeed, if you look at https://github.com/WebKit/WebKit/commit/1ce3567b28b06f9a26a94e0bffad3c9940b71c39 I started without it, then you can see all the bots failed in https://github.com/WebKit/WebKit/pull/40625#issuecomment-2715664378

Reply:
I think what's happening is that WritingTools is a private framework, and someone will need to add more to `WritingToolsSPI.h` to get this building publicly. We can't write USE_APPLE_INTERNAL_SDK here to make this more obvious, so probably fine to leave for now.